suite-native/atoms/src/Select/SelectItem.tsx

```diff
@@ -63,7 +65,7 @@ export const SelectItem = ({
 }: SelectItemProps) => {
     const { applyStyle } = useNativeStyles();
 
-    if (!value) return null;
+    if (G.isNull(value) || G.isUndefined(value)) return null;
```

Comment:
G.isNullable combines these two guards:
```suggestion
    if (G.isNullable(value)) return null;
```

Reply:
Nice. https://github.com/trezor/trezor-suite/pull/7883/commits/26d57a5343d89293463b6fe2ca52b7b7325e06ee